src/PasswordIndicatorGrid.js

```diff
@@ -2,28 +2,30 @@ import React from "react";
 
 import "./PasswordIndicatorGrid.css";
 import { StrengthIndicator } from "./components/PasswordStrengthIndicator";
+import zxcvbn from "./checkers/zxcvbn";
 
 function PasswordIndicatorGrid(props) {
   let indicators = [
     {
       name: "Basic Indicator",
       determineStrength: password => {
         password = password || "";
-        const strength = Math.min(password.length, 10);
-        let description = "weak";
+        const strength = Math.min(password.length + 1, 10);
+        let descriptionText = "weak";
         if (strength > 4 && strength < 7) {
-          description = "okay";
+          descriptionText = "okay";
         } else if (strength >= 7) {
-          description = "strong";
+          descriptionText = "strong";
         }
 
+        const description = <p>{descriptionText}</p>;
```

Comment:
changing `description` to a JSX element because that's more in line with the format that zxcvbn uses